app/controllers/orgs/lti_configurations_controller.rb

```diff
@@ -93,5 +94,11 @@ def ensure_current_lti_configuration
     def lti_configuration_params
       params.require(:lti_configuration).permit(:lms_link)
     end
+
+    def ensure_no_roster
+      return unless current_organization.roster
+      redirect_to edit_organization_path(current_organization),
+        alert: "LMS configuration failed as you already have a roster. Please delete roster and try again."
```

Comment:
See related comment for GClassroom

Reply:
I don't think we need to. Since we define what LMS is on the `#new` page is